bramkouwen/Neatly-In-Motion
# Updating CSS

## What does this PR do?
* It updates main.css

## How do you check this PR?
* Pull down the branch
* Check the added content to main.css

## Screenshots 
None.
